frontend/src/app/shared/components/loading-spinner/loading-spinner.component.html

```diff
@@ -1,12 +1,4 @@
-<div *ngIf="spinnerService.isLoading$ | async">
-    <div
-        [ngStyle]="{
-            'font-size': size,
-            margin: margin,
-            top: top,
-            left: left,
-            position: position
-        }"
```

Comment:
why did you remove this?

Reply:
styles are set in sass file 
why do we need this here?

Reply:
and because it is impossible to use our color-variables in ts

Reply:
in case when we have styles in a style file, this component is not reusable and cannot be customized for different needs

Reply:
ok, returned